airbyte-webapp/src/components/connection/CatalogTree/next/StreamDetailsPanel/StreamFieldsTable/SyncFieldCell/SyncFieldCell.tsx

```diff
@@ -58,20 +60,20 @@ export const SyncFieldCell: React.FC<SyncFieldCellProps> = ({
   }, [isCursor, isChildFieldCursor, isPrimaryKey, isChildFieldPrimaryKey, isNestedField, field.path]);
 
   return (
-    <>
+    <FlexContainer alignItems="center">
       {!showTooltip && (
```

Comment:
I think the distinction here is that selection could be disabled in readonly mode, but we wouldn't show a tooltip. So that's why just `isDisabled` or `selectionDisabled` didn't quite make sense to me!

Reply:
Oh, yep. That makes a lot of sense now, thanks!